pixsrl/opengl/entrypoint.vnc-virtual-gl

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# -*- shell-script -*-
+
+cat <<_XORG_CONF_EOF_ | tee /root/xorg.conf
+Section "Device"
+    Identifier     "Device0"
+    Driver         "nvidia"
+    VendorName     "NVIDIA Corporation"
+    BusID          "PCI:2:0:0"
```

Comment:
This should be converted to an environment variable with a sane default e.g. `PCI:1:0:0`